lib/datasource/maven/__snapshots__/index.spec.ts.snap

```diff
@@ -339,7 +339,7 @@ Array [
       "user-agent": "https://github.com/renovatebot/renovate",
     },
     "method": "GET",
-    "url": "https://repo.maven.apache.org/maven2/org/example/package/maven-metadata.xml",
+    "url": "http://repo.maven.apache.org/maven2/org/example/package/maven-metadata.xml",
```

Comment:
??

Reply:
Intentionally replaced `https` to `http` for this test. I needed to fix coverage and this test seemed pretty relevant for this.

Reply:
Instead, probably it would be better to remove HTTP support at all

Reply:
Wouldn't removing HTTP support break support for internal maven repositories that aren't served via HTTPS? 
Like sonatype nexus for example still comes out of the box with http as it's default port.
Other scenarios might see some old references to http:// where the server properly handles a 301 to https

Reply:
Yes, you're right